libs/perl.json

```diff
@@ -11,18 +11,23 @@
   {"id":"perl.cgi","imports":["CGI"],"name":"CGI","repo":"leejo/CGI.pm","tags":["cgi"],"tech":["internet"],"status":"awaiting-model"},
   {"id":"perl.furl","imports":["Furl"],"name":"Furl","repo":"tokuhirom/Furl","tags":["http-client"],"tech":["internet"],"status":"awaiting-model"},
   {"id":"perl.extutils-cbuilder","imports":["ExtUtils::CBuilder"],"name":"ExtUtils-CBuilder","repo":"Perl-Toolchain-Gang/ExtUtils-CBuilder","tags":["perl-c"],"tech":["interoperability"],"status":"awaiting-model"},
+  {"id":"perl.inline","imports":["Inline"],"name":"Inline","repo":"ingydotnet/inline-pm","tags":["perl-otherlangs"],"tech":["interoperability","https://github.com/ingydotnet/inline-pm"],"status":"awaiting-model"},
```

Comment:
remove the repo url from tech

Reply:
tags == inline-langs maybe?

Reply:
no perl prefix?

Reply:
I don't understand the question.

Reply:
I meant would it be better to have something like 'perl-inline-langs'?

Reply:
it's a good question, but I think the answer should be 'no'. We'll probably highlight language a library pertains to in a different way.